sktime/transformers/summarise.py

```diff
@@ -215,3 +218,72 @@ def transform(self, X, y=None):
         Xt = pd.DataFrame(Xt)
         Xt.columns = self.columns_
         return Xt
+
+
+class TsFreshTransfomer(BaseTransformer):
+
+    def get_time_series_container(self,X):
+        columns = X.columns
+        X_time_series = pd.DataFrame()
+        rows = X.shape[0]
+        
+        for i in range(rows):
+            column_index = 0
+            temp_dataframe = pd.DataFrame()
+            time_series_shape = X.iloc[i,0].shape
+            time_series_len = time_series_shape[0]
+            
+            series_time = np.arange(start=0,stop=time_series_len,step=1)
+            series_id = np.full(time_series_shape,(i+1))
+            
+            for j in range(len(columns)):
+                temp_dataframe[columns[column_index]] = X.iloc[i,j]
+                column_index += 1
+                
+            temp_dataframe['series_time'] = series_time
+            temp_dataframe['series_id'] = series_id
+    #         print(temp_dataframe.head())
+            X_time_series = X_time_series.append(temp_dataframe)
+    #     print(X_time_series.head())
+        return X_time_series
+        
+    
+    def get_formatted_predictions(self,y):
+        y_time_series_container = []
+        for i in range(len(y)):
+            y_time_series_container.append([(i+1),y[i]])
+        return y_time_series_container
+
+
+    
+    def __init__(self):
+        pass
+
+    def fit(X, y=None):
+        #empty
```

Comment:
For `fit_transform` will it be a better idea to use the `extract_relevant_features` from tsfresh as it performs the extraction, imputing and filtering at the same time. 

Reference from the  [Quick Start](https://tsfresh.readthedocs.io/en/latest/text/quick_start.html)
![rel_features](https://user-images.githubusercontent.com/29939762/73854547-a1498e00-482a-11ea-874f-45c6d7ff4a89.png)

Reply:
Yes, `extract_relevant_features()` is useful here, in terms of API design the questions is whether we want a single tsfresh transformer or one for extraction and one for extraction plus filtering, not sure how to best factor out the code

Reply:
Wouldn't it be confusing to have a single tsfresh transformer for both since just doing the transformation doesn't require the data frame to be fit before whereas if we have some other class called `TsFreshAugmentee` which fits then transforms and would also have a method called `fit_transform`? 

If you look at [tsfresh augmenter](https://github.com/blue-yonder/tsfresh/blob/48118627d9d4644906613e25b077ce2ec82ca2f9/tsfresh/transformers/relevant_feature_augmenter.py), they are using the tsfresh `feature_extractor` to transform first and then using `feature_selector` to fit and then in the `transform` method they require fit to have occurred first.

 Or is our `TsFreshTransformer` just required to extract the important features and return the data frame with only selected features as done by `extract_relevant_features`?

Reply:
Yes, having two transformers sounds good,
* `TsFreshFeatureExtractor` for feature extraction only, 
* `TsFreshRelevantFeatureExtractor`  for feature extraction and selection.